examples/pxScene2d/src/pxScene2d.cpp

```diff
@@ -906,6 +906,9 @@ rtError pxScene2d::createFontResource(rtObjectRef p, rtObjectRef& o)
 rtError pxScene2d::createScene(rtObjectRef p, rtObjectRef& o)
 {
   pxSceneContainer* sceneContainer = new pxSceneContainer(this);
+  if (mSceneContainer == NULL) {
+    mSceneContainer = sceneContainer;
```

Comment:
A Spark scene can create multiple child scenes.  This change will only store the first created scene container to mSceneContainer.  Also, can you please briefly describe the need to expose the first created scene container on pxScene2d?  Perhaps we can accomplish the ability you need Spark's existing APIs.

Reply:
Currently there is no interface to changeURL using the pxScriptView(). It support only the URL loading from the constructor. So I have to add one interface to meet this requirement :https://github.com/pxscene/pxCore/pull/1902/files#diff-34d65b1494b2c54a660cc9b8cda50fa4R3152. In such case, url can be changed using the sceneContainer of the scene of the pxScriptView. But scene also not saving the sceneContainer corresponding. So I have added this code to keep sceneContainer corresponding to the scene of the pxScriptView. Using this sceneContainer, I am able to reload the urls and do the suspend/resume of the reloaded url any times